src/wp-content/themes/twentytwentyfour/patterns/footer-portfolio.php

```diff
@@ -76,7 +76,7 @@
 			<!-- wp:group {"style":{"spacing":{"blockGap":"6px"}},"layout":{"type":"flex","flexWrap":"wrap"}} -->
 			<div class="wp-block-group">
 				<!-- wp:paragraph {"fontSize":"small"} -->
-				<p class="has-small-font-size"><?php esc_html_e( '&copy; 2024', 'twentytwentyfour' ); ?></p>
+				<p class="has-small-font-size"><?php esc_html_e( '&copy;', 'twentytwentyfour' ); ?></p>
```

Comment:
We should either use use `date( 'Y' );` in PHP here so that the date changes automatically (see [Twenty Twenty](https://github.com/WordPress/wordpress-develop/blob/a1852bd49b0995317ebdde8f2ee33eb818e8da9c/src/wp-content/themes/twentytwenty/footer.php#L21C9-L21C9), or remove it entirely. My understanding is that the copyright symbol needs to be accompanied by a date.

Reply:
I think it's better to just remove it entirely and leave the site title on its own